lib/connect-backbone-to-react.js

```diff
@@ -66,39 +68,26 @@ module.exports = function connectBackboneToReact(
     const displayName = getDisplayName(wrappedComponentName);
 
     class ConnectBackboneToReact extends Component {
-      constructor(props, context) {
-        super(props, context);
+      constructor(props) {
+        super(props);
 
-        this.setModels(props, context);
-
-        this.state = mapModelsToProps(this.models, this.props);
+        this.models = {};
+        this.state = {};
 
         this.createNewProps = this.createNewProps.bind(this);
+        this.renderChild = this.renderChild.bind(this);
 
         if (debounce) {
           const debounceWait = typeof debounce === 'number' ? debounce : 0;
           this.createNewProps = debounceFn(this.createNewProps, debounceWait);
         }
-
-        this.createEventListeners();
       }
 
-      setModels(props, context) {
-        const models = Object.assign({}, context.models, props.models);
+      setModels(models = {}) {
         validateModelTypes(models);
         this.models = models;
```

Comment:
I don't see this being updated ever either. I'm curious if we even need it anymore.